gtf/filter.go

```diff
@@ -59,3 +61,30 @@ func FilterVariantFiveUtr(v *vcf.Vcf, g map[string]*Gene, c map[string]*chromInf
 	a := FiveUtrBoolArray(g, c)
 	return VariantArrayOverlap(v, a)
 }
+
+func FindPromoter(genes []string, upstream int, downstream int, gtf map[string]*Gene, size map[string]chromInfo.ChromInfo) []bed.Bed {
+	var answer []bed.Bed
+	var currGene, transcript int
+	var name string
+	var trans *Transcript
+	var newBed = bed.Bed{Chrom: "", ChromStart: 0, ChromEnd: 0, FieldsInitialized: 4}
+
+	for currGene = range genes {
+		name = genes[currGene]
+		for currRecord := range gtf {
+			if gtf[currRecord].GeneName == name {
+				for transcript = range gtf[currRecord].Transcripts {
+					trans = gtf[currRecord].Transcripts[transcript]
+					if trans.Strand {
```

Comment:
Can trans.Strand can be checked like a bool? It seems like the lines in gtfFileTest.gtf all have + so I wonder

Reply:
Yeah, it is stored as a bool if you look at the gtf struct because . Isn't an option